### PR TITLE
docs: add more workflow sections to contributing.md

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ Added dev, test and lint section to contributing docs.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,9 +72,56 @@ Once approved, the author of the pull request,
...
 ### Testing
 
-// To do: Testing information
+Tests fall under unit, integration and end-to-end tests. To run all tests
+
+```sh
+$ npm test
+```
+
+To run tests for a specific package,
```
Comment (bmorelli25) [inline CONTRIBUTING.md]:
```suggestion
To run tests for a specific package:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -72,9 +72,56 @@ Once approved, the author of the pull request,
...
+
+```sh
+$ SCOPE=elastic-apm-js-base npm test
+```
+
+To run unit/integration/end-to-end tests for packages,
```
Comment (bmorelli25) [inline CONTRIBUTING.md]:
```suggestion
To run unit/integration/end-to-end tests for packages:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -72,9 +72,56 @@ Once approved, the author of the pull request,
...
+
+### Linting
+
+We use [prettier](https://github.com/prettier/prettier) for formatting code and [eslint](https://github.com/eslint/eslint) for other linting.
+
+```
+$ npm run lint
+```
+
+To fix formatting and other auto-fixable eslint errors,
```
Comment (bmorelli25) [inline CONTRIBUTING.md]:
```suggestion
To fix formatting and other auto-fixable eslint errors:
```

**Comment by vigneshshanmugam** [comment]
@bmorelli25 Thanks :) I have committed the suggestions.

**Pull request merged**